src/main/resources/templates/appellant_appeal_template.html

```diff
@@ -0,0 +1,542 @@
+                </div>
+
+            </dl>
+
+            <!-- MRN -->
+
+            <h1 class="heading-medium">MRN and appeal submission</h1>
+            <dl class="govuk-check-your-answers cya-questions-short">
+                <div>
+                    <dt class="cya-question">MRN originating office</dt>
+                    <dd class="cya-answer">{{ SyaCaseWrapper.mrn.dwpIssuingOffice | default('Not applicable') }}</dd>
+                </div>
+
+                <div>
+                    <dt class="cya-question">Date of decision</dt>
+                    <dd class="cya-answer">{{ SyaCaseWrapper.mrn.date | date(existingFormat="dd-MM-yyyy", format="dd MMMM yyyy") }}</dd>
+                </div>
+
+                <div>
+                    <dt class="cya-question">Date appeal submitted</dt>
+                    <dd class="cya-answer">{{ SyaCaseWrapper.mrn.dateAppealSubmitted| date(existingFormat="yyyy-MM-dd", format="dd MMMM yyyy") }}</dd>
+                </div>
+
+                <div>
+                    <dt class="cya-question">Why appeal is late</dt>
+                    <dd class="cya-answer">{{ SyaCaseWrapper.mrn.reasonForBeingLate | default('Not applicable') }}</dd>
+                </div>
+
+                <div>
+                    <dt class="cya-question">No MRN</dt>
+                    <dd class="cya-answer">{{ SyaCaseWrapper.mrn.reasonForNoMRN | default('Not applicable') }}</dd>
+                </div>
+
+            </dl>
+
+            <!-- Reasons for appealing -->
+
+            <h1 class="heading-medium">Reasons for appealing</h1>
+            <dl class="govuk-check-your-answers cya-questions-short">
+                <div>
+                    <dt class="cya-question">What you disagree with and why</dt>
+                    <dd class="cya-answer"><p>{{ SyaCaseWrapper.reasonsForAppealing.reasons }}</p></dd>
+                </div>
+
+                <div>
+                    <dt class="cya-question">Anything else you want to tell the tribunal</dt>
+                    <dd class="cya-answer">{{ SyaCaseWrapper.reasonsForAppealing.otherReasons }}</dd>
+                </div>
+
+            </dl>
+
+            <!-- The hearing -->
+
+            <h1 class="heading-medium">The hearing</h1>
+            <dl class="govuk-check-your-answers cya-questions-short">
+                <div>
+                    <dt class="cya-question">Attending the hearing</dt>
+                    <dd class="cya-answer">{{ SyaCaseWrapper.hearing.wantsToAttend ? "Yes" : "No" }}</dd>
+                </div>
+
+                <div>
+                    <dt class="cya-question">Dates you're not available</dt>
+                    <dd class="cya-answer">
+                        {% if SyaCaseWrapper.hearing.scheduleHearing == true  %}
+                        {% for date in SyaCaseWrapper.hearing.datesCantAttend %}
+                        {{ date }}<br/>
+                        {% endfor %}
+                        {% else %}
+
```

Comment:
Can you make sure the latest changes to the TYA API appellant_appeal_template.html are copied over?

Reply:
Not now later. I copied the latest template but the PDF generation was failing because of some exception in the date format.